client/src/main/java/org/hyperskill/database/clientside/Commands.java

```diff
@@ -0,0 +1,14 @@
+package org.hyperskill.database.clientside;
+
+import com.beust.jcommander.IParameterValidator;
+import com.beust.jcommander.ParameterException;
+
+public class Commands implements IParameterValidator {
+    public void validate(String name, String value)
+            throws ParameterException {
+        value = value.trim().toLowerCase();
```

Comment:
It's a bad practice to reassign input parameter
Better treat and mark them as final
More and more languages do that by default